curriculum/challenges/english/25-front-end-development/quiz-backend-javascript/66f1b11026394bcdb7c42473.md

```diff
@@ -17,375 +17,374 @@ Answer all of the questions below correctly to pass the quiz.
+Building and running JavaScript applications efficiently.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the Express method `app.use()` do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Handles GET requests
 
 ---
 
-Placeholder distractor 2
+Starts the server
 
 ---
 
-Placeholder distractor 3
+Mounts middleware functions
 
 #### --answer--
 
-Placeholder answer
+Mounts middleware functions
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default port for an Express application?
 
 #### --distractors--
 
-Placeholder distractor 1
+4000
 
 ---
 
-Placeholder distractor 2
+8080
 
 ---
 
-Placeholder distractor 3
+4001
 
 #### --answer--
 
-Placeholder answer
+3000
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the cors package in an Express application?
 
 #### --distractors--
 
-Placeholder distractor 1
+To compress response data
 
 ---
 
-Placeholder distractor 2
+To manage session storage
 
 ---
 
-Placeholder distractor 3
+To log HTTP requests
 
 #### --answer--
 
-Placeholder answer
+To enable Cross-Origin Resource Sharing
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which SQL command is used to delete records from a database?
 
 #### --distractors--
 
-Placeholder distractor 1
+DROP
 
 ---
 
-Placeholder distractor 2
+REMOVE
 
 ---
 
-Placeholder distractor 3
+PURGE
 
 #### --answer--
 
-Placeholder answer
+DELETE
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a popular ORM for Node.js?
 
 #### --distractors--
 
-Placeholder distractor 1
+Bun
 
 ---
 
-Placeholder distractor 2
+PostgreSQL
 
 ---
 
-Placeholder distractor 3
+V8
 
 #### --answer--
 
-Placeholder answer
+Mongoose
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the dotenv package in a Node.js application?
 
 #### --distractors--
 
-Placeholder distractor 1
+To manage sessions
 
 ---
 
-Placeholder distractor 2
+To optimize performance
 
 ---
 
-Placeholder distractor 3
+To compile TypeScript
 
 #### --answer--
 
-Placeholder answer
+To load environment variables from a .env file
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you retrieve query parameters in an Express route?
 
 #### --distractors--
 
-Placeholder distractor 1
+req.body
 
 ---
 
-Placeholder distractor 2
-
+req.data
 ---
 
-Placeholder distractor 3
+req.params
 
 #### --answer--
 
-Placeholder answer
+req.query
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to run scripts defined in the package.json file?
 
 #### --distractors--
 
-Placeholder distractor 1
+npm execute
 
 ---
 
-Placeholder distractor 2
+npm run-script
 
 ---
 
-Placeholder distractor 3
+npm launch
 
 #### --answer--
 
-Placeholder answer
+npm run
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which JavaScript runtime is designed to be lightweight and optimized for performance?
```

Comment:
Again, I'm not sure different Runtimes are really relevant to this quiz. It is about Node and Express.